### PR TITLE
Initial cmdo protoype

### PR DESCRIPTION
**Pull request by mattgi**

### CODE REVIEW

**Comment by Kodemon** [comment]
Looks shit, rejected.

**Comment by Kodemon** [comment]
Dude, how did you even

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,26 +1,17 @@
 {
+  "editor.formatOnSave": false,
```
Comment (Kodemon) [inline .vscode/settings.json:2]: Do you need the language specific format on save false if you have a global one?

Comment (mattgi) [reply .vscode/settings.json:2]: Probably not. Those settings were copy pasted from someone saying 'yay its working'. I can remove though.. easy.

**Pull request merged**